### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by Nid21cs**
Added in contributions in 
-  Team based work
- Add in work done in DG
- Add in code repo link

### CODE REVIEW

File: docs/team/nid21cs.md [hunk 1]
```diff
@@ -7,36 +7,76 @@ title: Nidhish Krishnan's Project Portfolio Page
...
+2. **Introduced `task` package :**
+   
+   - Contains classes which control tasks for Groups and students, e.g `Todo`, `Deadline`.
+   - Controls behaviour of the tasks itself
+   - Contains methods to manage and change each task as well as the taskList
+
+## User Guide Contributions
+**Non-Feature Aspects**
+
+  - How to use User Guide
```
Comment (mingyuanc) [inline docs/team/nid21cs.md:39]: Cap bro

File: docs/team/nid21cs.md [hunk 2]
```diff
@@ -7,36 +7,76 @@ title: Nidhish Krishnan's Project Portfolio Page
...
+
+## User Guide Contributions
+**Non-Feature Aspects**
+
+  - How to use User Guide  
+  - Exciting future developments
+
+**Feature Aspects**
+
+-  `todo` command
```
Comment (mingyuanc) [inline docs/team/nid21cs.md:44]: todo got a space before it

File: docs/team/nid21cs.md [hunk 3]
```diff
@@ -7,36 +7,76 @@ title: Nidhish Krishnan's Project Portfolio Page
...
-  * Contributed to forum discussions (examples: to be added soon)
-  * Reported bugs and suggestions for other teams in the class (examples: to be added soon)
-  * Some parts of the history feature I added was adopted by several other class mates (to be added soon)
+## Review and Mentoring Contributions
 
-* **Tools**:
-  * to be added soon
+Here are the pull requests I reviewed and provided valuable feedback and suggestions:
 
+- **[Pull Request #269](https://github.com/AY2324S1-CS2103T-W15-2/tp/pull/269)**
```
Comment (mingyuanc) [inline docs/team/nid21cs.md:81]: bro this cap, all u did was said my model was misspelt

**Review by mingyuanc — CHANGES_REQUESTED** [review]

File: docs/team/nid21cs.md [hunk 4]
```diff
@@ -7,36 +7,76 @@ title: Nidhish Krishnan's Project Portfolio Page
...
+   
+3. **Use cases**:
+   - Added use cases to show possible workflow.
 
-* **New Feature**: to be added soon.
+4. **Non-Functional Requirements**:
+   - Added non-functional requirements to the applications to describe our system capabilities and constraints.
 
-* **Code contributed**: to be added soon.
+5. **User stories**:
```
[the remark this answers is not in the document]
Comment (Nid21cs) [reply docs/team/nid21cs.md:63]: brother i add long time ago start of proj

**Review by mingyuanc — APPROVED** [review]
Apart from all the cap, LGTM

**Pull request merged**